Sources/xcodeproj/Errors/Errors.swift

```diff
@@ -161,12 +161,24 @@ enum PBXProjEncoderError: Error, CustomStringConvertible {
 enum PBXProjError: Error, CustomStringConvertible {
```

Comment:
I'd conform `Equatable` here so that we can compare the errors from the tests.

Reply:
Good point, done 👍